client/src/modules/templates/bhYesNoRadios.tmpl.html

```diff
@@ -1,7 +1,8 @@
-<div class="form-group" data-bh-yes-no-radios>
-  <label class="control-label" translate>
-    {{$ctrl.label}}
-  </label>
+<div class="form-group" ng-form="bhYesNoForm"
+  data-bh-yes-no-radios
+  ng-class="{ 'has-error' : bhYesNoForm.$submitted && bhYesNoForm.$invalid }">
+
+  <label class="control-label" translate>{{$ctrl.label}}</label>
```

Comment:
Excellent!  It will apply to all Yes/No radios in all forms, right?

Reply:
That is correct!